requirements.txt

```diff
@@ -12,13 +12,11 @@ sqlalchemy-postgres-copy==0.3.0
 networkx==1.11
 newrelic==2.96.0.80
 SQLAlchemy==1.0.17
-mandrill==1.0.57
 icalendar==3.9.1
 GitPython==1.0.1
 gunicorn==19.7.1
 gevent==1.2.2
 webargs==0.18.0
-slacker==0.8.6
```

Comment:
Just confirming that we don't need this anymore as we're just posting directly over HTTPS to Slack now?

Reply:
yea it was too easy to justify importing a dependency